packages/fluentui/react-northstar/src/components/SvgIcon/SvgIcon.tsx

```diff
@@ -13,6 +13,11 @@ import { ThemeContext } from 'react-fela';
 
 import { ProviderContextPrepared, WithAsProp, withSafeTypeForAs } from '../../types';
 
+export type SvgIconStylesProps = Pick<
+  SvgIconProps,
+  'bordered' | 'circular' | 'disabled' | 'outline' | 'rotate' | 'size' | 'xSpacing'
+>;
```

Comment:
```suggestion
export type SvgIconStylesProps = Required<Pick<
  SvgIconProps,
  'bordered' | 'circular' | 'disabled' | 'outline' | 'rotate' | 'size' | 'xSpacing'
>>;
```

And let's also use in `useStyles` generic: `useStyles<SvgIconStylesProps>`